src/trio/_subprocess.py

```diff
@@ -304,7 +304,7 @@ def kill(self) -> None:
 
 
 async def _open_process(
-    command: list[str] | str,
+    command: StrOrBytesPath | Sequence[StrOrBytesPath],
```

Comment:
Wondering, why is type var here called `StrOrBytesPath`? Wouldn't it be `StrOrBytes`, or is `StrOrBytesPath` special somehow?

Reply:
Because it's typed as `str | bytes | PathLike[str] | PathLike[bytes]`?

Reply:
Reading the docstring currently in place, I wasn't under the impression it supported pathlikes. If you could update that then, that would be great!

Reply:
The docstring atm distinguishes what is possible with `shell=True` and what isn't, but I think paths are fine either way. I can update the docstring though. (this also appears in the function signature Sphinx makes)

Reply:
I would seem to be for the best to mention that, yes. After that, I have no objections.